instant-xml/tests/forward-enum.rs

```diff
@@ -4,19 +4,19 @@ use similar_asserts::assert_eq;
 
 use instant_xml::{from_str, to_string, FromXml, ToXml};
 
-#[derive(Debug, Eq, FromXml, PartialEq, ToXml)]
+#[derive(Debug, PartialEq, FromXml, ToXml)]
```

Comment:
Sorry, one more tiny nit: prefer to keep these in alphabetic order, as before.

Reply:
It's okay, normally I do too, but grouped by those from std and then by each crate.

Fixed.

Reply:
Shame that order may sometimes matter, (bad derive macro implementation imo), otherwise it could be taken care of by rustfmt.